v2/internal/controllers/crd_compute_vm_20201201_test.go

```diff
@@ -98,10 +98,12 @@ func newVMNetworkInterface(tc *testcommon.KubePerTestContext, owner *genruntime.
 	}
 }
 
-func Test_Compute_VM_CRUD(t *testing.T) {
+func Test_Compute_VM_20201201_CRUD(t *testing.T) {
 	t.Parallel()
 
 	tc := globalTestContext.ForTest(t)
+	tc.AzureRegion = to.StringPtr("westeurope")
```

Comment:
Leave a note as to why this was done, and/or that it's temporary and presumably capacity related?